lrpar/src/lib/ctbuilder.rs

```diff
@@ -363,7 +363,9 @@ where
                         // Iterate over all $-arguments and replace them with their respective
                         // element from the argument vector (e.g. $1 is replaced by args[0]). At
                         // the same time extract &str from tokens and actiontype from nonterminals.
-                        outs.push_str(&format!("fn {prefix}action_{}({prefix}input: &str, {prefix}args: &Vec<AStackType<{actiont}, {}>>) -> {actiont} {{\n",
+                        outs.push_str(&format!("#[allow(clippy::let_and_return)]
```

Comment:
What is this `let_and_return` for? I can't see it being used here.

Reply:
Basically Clippy complains about generated functions that look like:

```rust
fn f() {
    let x = ...;
    x
}
```

and this turns that warning off.

Reply:
I see.